docs/modules/ROOT/pages/development.adoc

```diff
@@ -347,6 +347,34 @@ The corrector allows you to `insert_after`, `insert_before`, `wrap` or
 Range can be determined on `node.location` where it brings specific
 ranges for expression or other internal information that the node holds.
 
+==== Preventing Clobbering
+
+The corrector detects and prevents correcting overlapping nodes, to prevent one correction from clobbering another.
```

Comment:
Should we have an example of the error message?

Reply:
Always a good idea IMO.